Gemfile.lock

```diff
@@ -127,4 +127,4 @@ DEPENDENCIES
   tzinfo-data
 
 BUNDLED WITH
-   2.1.4
+   2.3.22
```

Comment:
This doesn't quite fit with the ruby version. Probably just needs to be left out.

Reply:
This output is automated with `bundle update`. Now I get `BUNDLED WITH  2.4.22` when I run `bundle update` (still using Ruby 2.7.7).